src/twisted/test/test_amp.py

```diff
@@ -2008,8 +2012,6 @@ def test_negotiationFailedByClosing(self):
         # reasonable.
         self.assertFailure(d, error.PeerVerifyError)
 
-    skip = skipSSL
```

Comment:
I cant add a comment for skipSSL definition... but I thin that line 35 to line 38 can be removed as skipSSL is no longer used ... or should no longer be used.

Reply:
skipSSL is still used in this file.  I didn't remove it here, because I have a separate PR where I used @skipIf to convert functions: https://github.com/twisted/twisted/pull/1264

Reply:
ok.